src/main/groovy/wooga/gradle/unity/tasks/SetAPICompatibilityLevel.groovy

```diff
@@ -23,7 +23,7 @@ class SetAPICompatibilityLevel extends ConventionTask {
 
     @Input
     @Optional
-    APICompatibilityLevel getApiCompatibilityLevel() {
+    Map getApiCompatibilityLevel() {
```

Comment:
I wonder if we actually would prefer a `Map<BuildTargetGroup, ApiCompatibilityLevel>` here. Since we have problems with `BuildTargetGroup` being not very portable to different Unity versions we could use `Map<String, ApiCompatibilityLevel>` that would be the most honest version.

Reply:
Refactored it to use Map<String, APICompLevel>